### ISSUE TITLE
Some new Faker.js helpers expect an object as the first parameter

### ISSUE THREAD

**Issue by 255kb** [mockoon/mockoon#1218]
Some new Faker.js helpers like [`date.alpha`](https://fakerjs.dev/api/string.html#alpha) expect an object as the first parameter: 
```
{
  length: {
    min: 1,
    max: 5
  }
}
```

We currently support ordered arguments: `{{faker 'helper.name' 5 6}}`, or named parameters: `{{faker 'helper.name' min=5 max=6}}`. 
Providing an object is possible but requires the use of the object helper, sometimes multiple times: 

```
{{faker 'string.alpha' (object length=(object min=20 max=20))}}
{{faker 'string.alpha' (object length=6 casing='upper')}}
```

We could try to find a way to convert named params to objects, maybe something like: 
`{{faker 'string.alpha' 'length.min'=20 'length.max'=20}}`

**Comment by 255kb** [comment]
@ajatkj sure! I will assign the task to you. 
I don't know if this can be handled in an automated way or if you have to create a list of helpers accepting an object only. Also, some of their functions like `date.alpha` still accept either a primitive as the first argument, or an object.

**Comment by 255kb** [comment]
📦 This is now released in [v6.2.0](https://mockoon.com/releases/6.2.0/).
